lib/msprime.c

```diff
@@ -5928,9 +5994,121 @@ msp_add_population_split(
     de->params.population_split.num_derived = num_derived;
     de->change_state = msp_population_split;
     de->print_state = msp_print_population_split;
+    /* Set the ancestral population state to not active before the event. */
+    self->initial_populations[ancestral].state = MSP_POP_STATE_INACTIVE;
```

Comment:
I'm a bit confused about why the initial state of the ancestral population is set to be inactive only at this point in the simulation. Once the split has been simulated, the ancestral population becomes active, so it's too late to find any bugs to do with the initial inactive state, right?

Reply:
This is done *before* the simulation is actually run. All the events are added before we run ``initialise()``, and run later.

Reply:
Ah, okay!